src/ChatGPTBrowserClient.js

```diff
@@ -178,6 +178,71 @@ export default class ChatGPTBrowserClient {
         return response;
     }
 
+    async deleteConversation(conversationID) {
```

Comment:
I'd prefer this to be `conversationId` instead of `conversationID` to match the other parts of the code.